src/main/java/seedu/duke/reflection/InvalidCommand.java

```diff
@@ -0,0 +1,21 @@
+package seedu.duke.reflection;
+
+import seedu.duke.exception.InvalidCommandException;
+
+/**
+ * Doesn't seem to align with what is required for command superclass
+ */
+public class InvalidCommand {
+    private static final String INVALID_COMMAND_MESSAGE = "Sorry this command is invalid.";
+
+    /**
+     * Throw error and alert users of invalid command.
+     *
+     * @throws InvalidCommandException Invalid command from users
+     */
+    public void alertError() throws InvalidCommandException {
+        throw new InvalidCommandException(INVALID_COMMAND_MESSAGE);
+    }
+}
+
+
```

Comment:
Is this file really needed? I don't understand the benefit of having a wrapper here. 
Why cant the offending code `ReflectionManager, line 209` just throw the error direct there?